com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs

```diff
@@ -9,7 +9,7 @@ public class PlayerInitializer: MonoBehaviour
         void Start()
         {
             // Initializing the start position of the player.
-            transform.position = new Vector3(CinemachineBrain.ActiveBrainCount * 2, 2, 0);
+            transform.position = new Vector3(PlayerCounter.PlayerCount * 2, 2, 0);
 
             // Shift one bit per brain Count.
             m_CinemachineBrain.ChannelMask = (OutputChannel.Channels)(1 << CinemachineBrain.ActiveBrainCount);
```

Comment:
why not use PlayerCounter here too?

Reply:
Could make the script more reusable outside of the context of the sample. For example if there is a different number of brain. Do you want the change? I don't mind restraining the usage.

Reply:
I think it's confusing this way.  Let PlayerCounter be the single source for player count.

Reply:
Fixed